R/mixing.R

```diff
@@ -0,0 +1,72 @@
+create_transmission_mixer <- function(
+  variables,
+  parameters,
+  lagged_eir,
+  lagged_infectivity,
+  mixing_tt,
+  export_mixing,
+  import_mixing,
+  p_captured_tt,
+  p_captured,
+  p_success
+  ) {
+  function (timestep) {
+    n_pops <- length(variables)
+    rdt_positive <- vnapply(
+      seq(n_pops),
```

Comment:
```suggestion
      seq_len(n_pops),
```

`seq` behaves surprisingly when given a size of 0. Admittedly this is unlikely to happen, but safe to just use `seq_len`